src/core/qgis.h

```diff
@@ -53,7 +53,8 @@ class CORE_EXPORT QGis
     //! Used for symbology operations
     // Feature types
     // @deprecated use QgsWKBTypes::Type
-    /* Q_DECL_DEPRECATED */ enum WkbType
+    /* Q_DECL_DEPRECATED */
+    enum WkbType
```

Comment:
Could someone please confirm the `Q_DECL_DEPRECATED` still applies to `WkbType`?